Formula/c/cabin.rb

```diff
@@ -47,6 +47,9 @@ class Cabin < Formula
   end
 
   def install
+    # Allow usage of fmt 12.
+    # TODO: Upstream this.
+    inreplace "Makefile", "fmt < 12.0.0", "fmt < 13.0.0"
```

Comment:
it builds fine for me, so I submitted the patch, https://github.com/cabinpkg/cabin/pull/1231